baybe/recommenders/meta/base.py

```diff
@@ -53,14 +54,16 @@ def select_recommender(
         batch_size: int = 1,
         train_x: Optional[pd.DataFrame] = None,
         train_y: Optional[pd.DataFrame] = None,
-    ) -> Recommender:
+    ) -> PureRecommender:
         """Select a recommender for the given experimentation context.
```

Comment:
Change Docstring to "Select a pure recommender"

Reply:
changed